code/game/machinery/computer/ai_core.dm

```diff
@@ -33,7 +33,7 @@
 			if(isWelder(P))
 				if(P.use_tool(user, src, WORKTIME_NORMAL, QUALITY_WELDING, FAILCHANCE_NORMAL))
 					to_chat(user, "<span class='notice'>You deconstruct the frame.</span>")
-					new /obj/item/stack/material/plasteel( loc, 4)
```

Comment:
i'm assuming you checked the craft menu to see what these should cost?

Reply:
![image](https://user-images.githubusercontent.com/49106866/106011628-efe6dc80-60ba-11eb-83a9-c0283409acad.png)